crates/swc_ecma_minifier/tests/terser/compress/blocks/issue_1672_for_strict/output.js

```diff
@@ -1,12 +1,10 @@
 "use strict";
-switch (
-    function () {
-        return xxx;
-    }
-) {
+switch(function() {
```

Comment:
Seems like terser does not implement annex B?

Reply:
It does. Terser just don't rename ident unnecessarily

Reply:
try adding 'use strict' to following code in https://try.terser.org/
```js
switch (
    function () {
        return xxx;
    }
) {
    case xxx:
        for (; console.log("FAIL"); ) {
            function xxx() {}
        }
        break;
}
```
you could see function xxx() {} be removed or not